test/const.test.js

```diff
@@ -65,3 +145,53 @@ test('schema with const and invalid object', (t) => {
     t.ok(err)
   }
 })
+
+test('schema with const and invalid number', t => {
```

Comment:
Let me make an example @Uzlopak 

talking about the behaviour, I think this test should be always successful:

```
const x = stringify({
      foo: 2
    })
// x should be '{"foo":1}'
```

No matter what the input object is, the output should be always equals to `const`